app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -47,9 +47,28 @@ export function SelfieStepWithHeader({ defaultSideProps, selfieValue }) {
   );
 }
 
-/**
- * @typedef {'front'|'back'} DocumentSide
- */
+export function DocumentFrontAndBackCapture({ defaultSideProps, value }) {
+  /**
+   * Sides of the ID document to present as file input.
+   *
+   *
+   * @typedef {'front'|'back'} DocumentSide
+   * @type {DocumentSide[]}
+   */
+  const documentsSides = ['front', 'back'];
```

Comment:
what if we converted this file to `.tsx` so we can keep these defintions in the TS syntax?

Reply:
For the purposes of this PR I'd rather keep this file as a `.jsx`, but:
- I agree, this should be a `.tsx` file.
- We are hoping to slowly convert the remaining `.jsx` files in the `document-capture` package to `.tsx`
- I'm hesitating to do this now for two reasons:
    1.  This cleanup work is already a stretch to include with the PR I'm working on about selfie errors.
    2. I wouldn't want to change the components -and- convert to a `.tsx` file in one PR. The diff would be very hard to read and I think that makes a mistake/regression much more likely.

Reply:
I'm looking at the file and I'm only seeing two blocks of JSDoc definitions, I think the diff converting to TSX would be minimal!

Reply:
Oh, interesting. I meant the diff that happens when you rename a file (from `.jsx` to `.tsx` because git thinks you're deleting one file and creating another.

Reply:
I just double checked and the diff does get huge b/c git doesn't really understand 'renaming' files.

Reply:
I just converted it as an exercise, this is all I needed, and `tsc` passed. Git even followed the file rename

```diff
diff --git a/app/javascript/packages/document-capture/components/documents-step.jsx b/app/javascript/packages/document-capture/components/documents-step.tsx
similarity index 82%
rename from app/javascript/packages/document-capture/components/documents-step.jsx
rename to app/javascript/packages/document-capture/components/documents-step.tsx
index fe2002f0f..627d67da6 100644
--- a/app/javascript/packages/document-capture/components/documents-step.jsx
+++ b/app/javascript/packages/document-capture/components/documents-step.tsx
@@ -1,6 +1,10 @@
 import { useContext } from 'react';
 import { useI18n } from '@18f/identity-react-i18n';
-import { FormStepsButton, FormStepsContext } from '@18f/identity-form-steps';
+import {
+  FormStepComponentProps,
+  FormStepsButton,
+  FormStepsContext,
+} from '@18f/identity-form-steps';
 import { PageHeading } from '@18f/identity-components';
 import { Cancel } from '@18f/identity-verify-flow';
 import HybridDocCaptureWarning from './hybrid-doc-capture-warning';
@@ -47,36 +51,24 @@ export function SelfieStepWithHeader({ defaultSideProps, selfieValue }) {
   );
 }
 
-/**
- * @typedef {'front'|'back'} DocumentSide
- */
+type DocumentSide = 'front' | 'back';
 
-/**
- * @typedef DocumentsStepValue
- *
- * @prop {Blob|string|null|undefined} front Front image value.
- * @prop {Blob|string|null|undefined} back Back image value.
- * @prop {Blob|string|null|undefined} selfie Selfie image value.
- * @prop {string=} front_image_metadata Front image metadata.
- * @prop {string=} back_image_metadata Back image metadata.
- */
+interface DocumentsStepValue {
+  front: Blob | string | null | undefined;
+  back: Blob | string | null | undefined;
+  selfie: Blob | string | null | undefined;
+  front_image_metadata?: string;
+  back_image_metadata?: string;
+}
 
-/**
- * @param {import('@18f/identity-form-steps').FormStepComponentProps<DocumentsStepValue>} props Props object.
- */
 function DocumentsStep({
   value = {},
   onChange = () => {},
   errors = [],
   onError = () => {},
   registerField = () => undefined,
-}) {
-  /**
-   * Sides of the ID document to present as file input.
-   *
-   * @type {DocumentSide[]}
-   */
-  const documentsSides = ['front', 'back'];
+}: FormStepComponentProps<DocumentsStepValue>) {
+  const documentsSides: DocumentSide[] = ['front', 'back'];
 
   const { t } = useI18n();
   const { isMobile } = useContext(DeviceContext);
```

Reply:
I can push these changes to your branch if you'd like!

Reply:
Oh thanks! I'm surprised and pleased. I'd love if you pushed those, appreciate the demo/fix.

Reply:
done! in fe34c984f3b30703df19cc090c068ce4af8e23e9